astroid/brain/brain_namedtuple_enum.py

```diff
@@ -399,6 +399,21 @@ def infer_typing_namedtuple_class(class_node, context=None):
     return iter((generated_class_node,))
 
 
+def infer_typing_namedtuple_function(node, context=None):
```

Comment:
Are we keeping the compat to 3.5 ? If that's not the case we could add typing on new functions :)